game/hud/package.json

```diff
@@ -92,7 +92,7 @@
     "@types/d3-ease": "^1.0.7",
     "@types/enzyme": "2.8.0",
     "@types/isomorphic-fetch": "0.0.33",
-    "@types/lodash": "^4.14.66",
+    "@types/lodash": "4.14.66",
```

Comment:
is there a conflict with newer patch versions of lodash?

Reply:
no really a *conflict*, the newer types for lodash add much better type support, which is causing loads of type errors with the current lodash usage. This could be addressed in a separate PR, or I can upgrade the types here and fix all the issues.

Reply:
ok, we should probably just make an issue for this to be addressed in a subsequent PR then.